src/igvfd/types/file.py

```diff
@@ -629,13 +629,19 @@ def content_summary(self, principal_dimension, secondary_dimensions, content_typ
             'notSubmittable': True,
         }
     )
-    def summary(self, request, content_summary, file_set):
+    def summary(self, request, content_summary, file_set, filtered=None):
         file_set_object = request.embed(file_set, '@@object_with_select_calculated_properties?field=@type')
         predicted = None
         if 'PredictionSet' in file_set_object['@type']:
             predicted = 'predictive'
+        if filtered is True:
+            filtered_phrase = 'filtered'
+        elif filtered is False:
+            filtered_phrase = 'unfiltered'
+        else:
+            filtered_phrase = 'unfiltered'
```

Comment:
Since there are only two cases, I think this section can be simplified somewhat. Something like: 
```
filtered_phrase = 'unfiltered'
if filtered:
   filtered_phrase = 'filtered'
````
 This can also apply to the summary property for Tab File

Reply:
Updated